mds/management/commands/poll_providers.py

```diff
@@ -0,0 +1,118 @@
+import datetime
+import logging
+import traceback
+import urllib.parse
+
+from django.contrib.gis import geos
+from django.core import management
+from django.db.models.aggregates import Max
+
+import requests
+import pytz
+
+from mds import models
+
+
+NOT_PROVIDED = object()
+
+logger = logging.getLogger(__name__)
+
+
+class Command(management.BaseCommand):
+    help = "Poll providers to fetch their latest data."
+
+    def handle(self, *args, **options):
+        for provider in models.Provider.objects.all():
+            if not provider.base_api_url:
+                self.stdout.write(
+                    "Provider %s has no URL defined, skipping." % provider.name
+                )
+                continue
+
+            self.stdout.write("Polling %s... " % provider.name, ending="")
```

Comment:
maybe use `logger.info` instead of `stdout` for easier integration in third party softwares